### ISSUE TITLE
Enable 'ems off' as in current original DosBox repo?

### ISSUE THREAD

**Issue by Workings** [joncampbell123/dosbox-x#1819]
Could you recreate the 'ems off/on' commands as in the latest versions of DosBox? Allows for the like of Ultima 7 to be run from a batch file rather than having to change config files each time.

**Comment by rderooy** [comment]
DOSBox-X decided not to implement it the same way as DOSBox, as it causes namespace conflicts with DOS programs.

Simply run it as follows;

```config -set ems false```

This also works with regular DOSBox.

**Comment by Workings** [comment]
Thanks. So take it no way to turn on/off within DosBox-x dos command prompt without rebooting?

**Comment by rderooy** [comment]
What reboot? "config" is a command on the Z:\ drive, both in vanilla DOSBox and DOSBox-X

![command_005](https://user-images.githubusercontent.com/17701098/91836289-db590200-ec4a-11ea-9620-8691252b5742.png)

**Comment by Workings** [comment]
As that only sets the config file? You need to restart dosbox-x for the change to take effect.

**Comment by rderooy** [comment]
No, "config" does not save it to the config file unless you tell it to.

**Comment by Workings** [comment]
So issue is if I run these commands on DosBox a non-ems game works:
ems off
ultima7
ems on

If do same with 'config -set ems false' the game complains it cannot run with ems on:
config -set ems false
ultima7 
[does not load and complains of ems on]

**Comment by Wengier** [comment]
@Workings For the current DOSBox-X version you can try this (for now):

dosbox-x -set ems=false

Without the need to change the config file. Support for dynamic change of EMS memory at runtime may be added in the next version.

**Comment by Wengier** [comment]
I have already added a new menu "Expanded memory (EMS)" (under "DOS") which allows you to dynamically enable or disable the EMS memory at run-time in addition to at start. You can also use CONFIG command to do so, e.g.

config -set ems=0
